tests/instructions/is_eq_neq.rs

```diff
@@ -27,7 +27,7 @@ mod is_eq_tests {
         let (function_variables, _proof) =
             vmtropy::execute_function(&program, function_name, &user_inputs).unwrap();
 
-        let expected_function_variables = vec!["r0", "r1", "r2"];
+        let expected_function_variables = vec!["r0", "r1", "r2", "r3"];
```

Comment:
The reason for the tests failing is because of this. In the function variables map we also have the constants as elements of the map.

One of this should fix the `u8` related tests (let me know if it doesn't):
 ```rust
let expected_function_variables = vec!["r0", "r1", "r2", "r3", "10u8"];
let expected_function_variables = vec!["r0", "r1", "10u8", "r2", "r3"];
```